src/main.js

```diff
@@ -449,51 +452,58 @@ function TimekitBooking() {
 
     $.extend(true, args, config.timekitCreateBooking);
 
-    if (config.timekitCreateEvent) {
-      $.extend(true, args.event, config.timekitCreateEvent); // backwards compatibility
-      utils.logDeprecated('config key "timekitCreateEvent" has been replaced, use "timekitCreateBooking" and see docs');
-    }
-
     utils.doCallback('createBookingStarted', config, args);
-    utils.doCallback('createEventStarted', config, args, true);
 
     var requestHeaders = {
       'Timekit-OutputTimestampFormat': 'Y-m-d ' + config.localization.emailTimeFormat + ' (P e)'
     };
 
     return timekit
     .headers(requestHeaders)
+    .include('attributes')
     .createBooking(args);
 
   };
 
   // Render the powered by Timekit message
   var renderPoweredByMessage = function(pageTarget) {
 
+    var campaignName = 'widget'
+    var campaignSource = window.location.hostname.replace(/\./g, '-')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'campaignName'.](https://www.codacy.com/app/laander/booking-js/file/2419896676/issues/source?bid=2969108&fileBranchId=3228106#l473)